salt/bugs/init.sls

```diff
@@ -4,18 +4,20 @@ include:
   - tls.lego
   - nginx
 
-/etc/consul.d/service-bugs.json:
+{% for tracker, config in pillar["bugs"]["trackers"].items() %}
+/etc/consul.d/service-{{ tracker }}.json:
```

Comment:
does this need to be service-{{tracker or just `{{ tracker }}` for full 1:1 mapping into haproxy?